Allow to move clock pins (#1172)
thewca/wca-regulations: wca-guidelines.md

## <article-12><notation><notation> Article 12: Notation

- 12a2+) [CLARIFICATION] For example, both Rw and 2Rw are valid notation for the same physical move for the 3x3x3 Cube. 1Rw is not valid notation for any NxNxN Cube, and 3Rw is not valid notation for the 2x2x2 Cube or the 3x3x3 Cube (but 3Rw is valid notation for the 4x4x4 Cube and larger NxNxN Cubes).

> - 12g2+) [REMINDER] In the past, pins notation was considered to be moves. Pins notation now can be applied during inspection period.

Reviewer: ```suggestion
- 12g2+) [REMINDER] In the past, the penalty for changing state of the pins in inspection was a DNF. This is no longer the case. 
```
I don't think "pins notation" is correct wording and it doesn't make much sense to me. I think this is much clearer

Author: ```suggestion
- 12g2+) [REMINDER] In the past, the penalty for changing state of the pins in inspection resulted in a DNF. This is no longer the case.
```

Maybe this?

> - 12g2+) [REMINDER] In the past, these notations were considered to be moves. Pin notation now can be applied during inspection period.

Author: ```suggestion
- 12g2+) [REMINDER] In the past, pins notation was considered to be moves. Pins notation now can be applied during inspection period.
```

- E3d++) [CLARIFICATION] Competitors should not consider a personal stopwatch or watch as the official time, and must submit their solution when the judge calls "STOP".

## <article-F><clock><clocksolving> Article F: Clock Solving

> - F3+) [REMINDER] In the past, the competitor had to place the puzzle on a mat in a standing position. This is no longer required.

Reviewer 2: ```suggestion
- F3+) [REMINDER] In the past, the competitor had to place the puzzle onto the mat in a standing position. This is no longer required.
```
